### ISSUE TITLE
add BLOCKED state for results

### ISSUE THREAD

**Issue by bonzini** [patchew-project/patchew#110]
When a test is failing on master, currently it will end up failing on all submitted patches too. We should instead mark the results as BLOCKED rather than PENDING in the series; the test can be triggered manually on the series that fixes the breakage.

This way: 1) we can make the "T" icon yellow instead of red; 2) we don't inundate the mailing list with failures; 3) we can easily search the affected patches and retrigger testing on them (with "blocked:testing" as a search term).

**Comment by famz** [comment]
Related: https://github.com/patchew-project/patchew/issues/32

**Comment by famz** [comment]
If all series are still tested regardless of their base (master) failure, the "fix" for the base breakage can be verified without human involvement.

All that is required is to turn "errors" into "warnings" when failures are also seen on a series "base", I think.

**Comment by bonzini** [comment]
True. So another possibility is to add a `base_status` to the testing result data, and use it to distinguish error (red) from warning (yellow) in the dashboard.
